api/app.ts

```diff
@@ -20,6 +20,8 @@ app.use(
 )
 
 // GET endpoint for the root
+app.get('/appointments', require('./appointments.ts'));
```

Comment:
This needs to be switched to the new style as in https://github.com/movshov/VA-Audiology-App/pull/10